src/UserGuide/Master/Reference/ConfigNode-Config-Manual.md

```diff
@@ -33,31 +33,31 @@ The environment configuration file is mainly used to configure the Java environm
 
 The details of each parameter are as follows:
 
-* MAX\_HEAP\_SIZE
+* MEMORY\_SIZE
 
-|Name|MAX\_HEAP\_SIZE|
+|Name|MEMORY\_SIZE|
```

Comment:
all of the changes need to clarify these parameters' former name. e.x. Former Name: MAX_HEAP_SIZE